### PR TITLE
[64] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by bint-saad**
[64] Style the Product Details Page using CSS
[64] Style the Product Details Page using CSS
3c
3a
3b

### CODE REVIEW

**Comment by adaugochi** [comment]
Kindly send screenshots for desktop and mobile phone view

File: submissions/hadizaSaadu/HTML/product-page.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <p><strong>Price: #15,000</strong></p>
+            <p> Shipping Cost: #5,000</p>
+            <p>Total: #20,000</p>
+            <button type="button" name="Submit" class="buy">Buy Now</button>
+            
+          </td>
+        </tr>
+        <tr>
+          <td colspan="2" >
+
```
Comment (Ireti203) [inline submissions/hadizaSaadu/HTML/product-page.html:39]: Remove whitespace

File: submissions/hadizaSaadu/HTML/product-page.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          <p class="desc" >Description</p>
+             <p class="desc2"> This piece has been carefully designed to suit the need of all
+              modesty lovers,<br />it is a free size and fits for all
+              occassions, do not miss out on this wonderful offer!!!
+            </p>
+          </td>
+          </tr>
+      </table>
+    </section>
+<table class="downtb" >
```
Comment (Ireti203) [inline submissions/hadizaSaadu/HTML/product-page.html:49]: Use a descriptive name for classes. "downtb" is not descriptive

File: submissions/hadizaSaadu/HTML/product-page.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          </td>
+          <td class="text1"> 
+            <p>GOLD STONED BLACK ABAYA</p>
+            <p><strong>Brand: Trish Collection</strong></p>
+            <p>Quantity: <input type="text" class="quantity" value="2"> </p>
+            <p><strong>Price: #15,000</strong></p>
+            <p> Shipping Cost: #5,000</p>
+            <p>Total: #20,000</p>
+            <button type="button" name="Submit" class="buy">Buy Now</button>
+
```
Comment (Ireti203) [inline submissions/hadizaSaadu/HTML/product-page.html:34]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by bint-saad** [comment]
tab3
desktop1
desktop2
mobile1
mobile2
mobile3
tab1
tab2

**Comment by bint-saad** [comment]
screenshot added........... thanks

**Review by adaugochi — APPROVED** [review]

**Pull request merged**